### ISSUE TITLE
python 3.10 not detected

### ISSUE THREAD

**Issue by czanik** [syslog-ng/syslog-ng#3709]
While trying to compile the latest syslog-ng git master snapshot on ```Fedora Rawhide```, detection of Python failed. It seems, that they upgraded to ```Python 3.10```, and ./configure cannot detect it:
```
checking for libsystemd >= 245... yes
checking for 3.10-embed... no
checking for 3.10... no
configure: error: Could not find the requested Python development libraries
```
```sudo``` resolved a similar problem recently

**Comment by Barkodcz** [comment]
Hi,

created a bugfix for this issue. Until then can be solve the issue by using "--with-python=python-3.10" instead of "--with-python=3.10". After the PR is merged can be used agian "--with-python=3.10".

**Comment by MrAnno** [comment]
Fixed by #3713.
